api/models.py

```diff
@@ -45,6 +45,7 @@ class AssetMake(models.Model):
     make_label = models.CharField(max_length=40, null=False)
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
     last_modified_at = models.DateTimeField(auto_now=True, editable=False)
+    asset_type = models.ForeignKey(AssetType, on_delete=models.CASCADE)
```

Comment:
i'm thinking we should not allow someone to delete a type that is referenced somewhere to avoid accidental deletions. maybe present the user a message to delete all associated makes first?

Reply:
Yes, makes sense. Would both help ?: A message indicating that all makes associated with that type will be delete and `on_delete=models.CASCADE` on the model ?

If we go with another option like `on_delete=models.SET_NULL` wont that have probably unnecessary records relating to nothing in the database ?